wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
 
 #### All
 
-| Message | Explanation | Solution |
-| :-- | :-- | :-- |
-| No problems were found in this map! | AiMod couldn't find any issues with your map! | You could upload your beatmap using BSS to allow osu!modders to find other mistakes and issues with your beatmap. |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- | :-- |
+| No problems were found in this map! | *Self-explanatory* | Your beatmap is good to go! You may [upload your beatmap using BSS](/wiki/Beatmapping/Beatmap_submission) to allow modders to find other mistakes and issues with your beatmap. |  |
```

Comment:
i think it would be better to just have these be empty in this case

```suggestion
| No problems were found in this map! |  | Your beatmap is good to go! You may [upload your beatmap using BSS](/wiki/Beatmapping/Beatmap_submission) to allow modders to find other mistakes and issues with your beatmap. |  |
```

Reply:
hmm i'm going to keep this one i think, without the \*self-explanatory\* disclaimers the page doesn't sit quite right for me :(